test/isonline.js

```diff
@@ -138,9 +160,10 @@ lab.experiment('isonline', () => {
         assert.ok(/online/.test(args[1]), 'returns online for online nodes');
         done();
       };
-      nmo.load({nmoconf: __dirname + '/fixtures/randomini'})
+      nmo.load({nmoconf: __dirname + '/fixtures/randomini', json: false})
         .then(() => {
           cli(common.NODE);
+          done();
```

Comment:
with the done()added there this async tests ends too early. if you take a look at the block above, line 159 - where i monkey patch console.log to verify that we are logging the result to the console and then call done - that is the asserting part of the test

i think that is the reason why test coverage went down